types/spreedly-iframe-browser/index.d.ts

```diff
@@ -44,6 +44,10 @@ declare namespace spreedly {
     interface InitOptions {
         numberEl: string;
         cvvEl: string;
+        nonce?: string;
+        certificateToken?: string;
+        signature?: string;
+        timestamp?: string;
```

Comment:
Based on Spreedly's [documentation](https://developer.spreedly.com/docs/iframe-api-lifecycle#security-requirements), it seems like this options are now required.

```suggestion
        nonce: string;
        certificateToken: string;
        signature: string;
        timestamp?: string;
```

Reply:
Yeah that is true, I wanted it to be compatible since right now before June 1st it is optional

Reply:
Done

Reply:
Ohh got it, I didn't know that 👍🏼